ConanToolWindowControl.xaml.cs

```diff
@@ -257,7 +269,127 @@ private void ShowConfigurationDialog()
             ThreadHelper.ThrowIfNotOnUIThread();
             _dte.ExecuteCommand("Tools.Options", GuidList.strConanOptionsPage);
         }
-        
+
+        private bool IsConanInitialized()
+        {
+            // TODO: What logic should go here?
+            return false;
+        }
+
+        private bool IsFileCommentGuarded(string path)
+        {
+            if (File.Exists(path))
+            {
+                string[] guardComment = _modifyCommentGuard.Split('\n');
+                string[] fileContents = File.ReadAllLines(path);
+                if (fileContents.Length > guardComment.Length && fileContents.AsSpan(0, guardComment.Length) == guardComment)
+                {
+                    return true;
+                }
+            }
+            return false;
+        }
+
+        private void WriteConanfileIfNecessary(string projectDirectory)
+        {
+            string path = Path.Combine(projectDirectory, "conanfile.py");
+            if (!IsFileCommentGuarded(path))
+            {
+                StreamWriter conanfileWriter = File.CreateText(path);
+                conanfileWriter.Write(_modifyCommentGuard +  "\n");
+
+                conanfileWriter.Write("from conan import ConanFile\n" +
+                    "class Package(ConanFile):\n" +

```

Comment:
I think we can add the full conanfile here, I will remove the hardcoded generator from the conan install call, something like this?

# This file is managed by Conan, contents will be overwritten.
# To keep your changes, remove these comment lines, but the plugin won't be able to modify your requirements

```python
from conan import ConanFile
from conan.tools.microsoft import vs_layout, MSBuildDeps

class ConanApplication(ConanFile):
    package_type = "application"
    settings = "os", "compiler", "build_type", "arch"

    def layout(self):
        vs_layout(self)

    def generate(self):
        deps = MSBuildDeps(self)
        deps.generate()

    def requirements(self):
        requirements = self.conan_data.get('requirements', [])
        for requirement in requirements:
            self.requires(requirement)

```